pkg/mgmt/devicenode/devicenode.go

```diff
@@ -0,0 +1,308 @@
+/*
+ Copyright © 2021 The OpenEBS Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package devicenode
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	k8serror "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+
+	apis "github.com/openebs/device-localpv/pkg/apis/openebs.io/device/v1alpha1"
+	"github.com/openebs/device-localpv/pkg/builder/nodebuilder"
+	"github.com/openebs/device-localpv/pkg/device"
+	"github.com/openebs/device-localpv/pkg/equality"
+)
+
+func (c *NodeController) listDeviceNames() ([]apis.Device, error) {
+	return device.GetDiskList()
```

Comment:
I don't see GetDisklist return disk size info? how GetCapacity can use this if that is not populated?